### ISSUE TITLE
Trying to use forward AD with native_dropout that does not support it

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/functorch#628]
Running [functional_autograd_benchmark](https://github.com/pytorch/pytorch/blob/58f78ff4e08a6d6a1fc0844dd19bb92fb139bbac/benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py) with functorch ([code](https://github.com/vfdev-5/pytorch/blob/add-functorch-bench/benchmarks/functional_autograd_benchmark/functional_autograd_benchmark.py)

Task `jvp` raises the error: `Trying to use forward AD with native_dropout that does not support it`

```
python functional_autograd_benchmark.py --model-filter=fcn_resnet --task-filter=jvp

Found functorch
Results for model fcn_resnet on task jvp: 2.09275484085083s (var: 1.481831532146316e-05)
Failed model using Functorch: fcn_resnet, task: jvp, Error message: 
         Trying to use forward AD with native_dropout that does not support it.
```

This is due to `nn.Dropout` used in FCNHead:

https://github.com/pytorch/pytorch/blob/58f78ff4e08a6d6a1fc0844dd19bb92fb139bbac/benchmarks/functional_autograd_benchmark/torchvision_models.py#L345

**Comment by zou3519** [comment]
Discussed offline -- PyTorch forward-mode AD doesn't support dropout yet
